fire/helputils.py

```diff
@@ -126,7 +126,8 @@ def HelpString(component, trace=None, verbose=False):
   return '\n'.join(lines)
 
 
-def _UsageStringFromFnDetails(command, args, varargs, keywords, defaults):
+def _UsageStringFromFnDetails(command, args, varargs, varkw, defaults,
```

Comment:
For the name of the variable, I think there is virtue in trying to adhere to the conventions of the inspect module, which keeps argspec as you currently use it, and signature as the ordered dictionary of arguments I described above.